pulp_smash/tests/rpm/api_v2/test_remove_unit.py

```diff
@@ -0,0 +1,283 @@
+# coding utf-8
+"""Test the functionality of syncing RPM repos with remove-unit option set.
+
+Following steps are executed in order to test correct functionality
+of repository created with valid feed and remove_missing option set.
+
+1. Create repository foo with valid feed, run sync, add distributor to it
+   and publish over http and https (RemoveMissingTestCase).
+2. Create second repository, bar, with feed pointing to first repository and
+   set remove_missing=True (RemoveMissingTrueTestCase) and remove_missing=False
+   respectively (RemoveMissingFalseTestCase) and run sync on them.
+3. Assert that repositories contain same set of units.
+4. Remove random unit from repository foo and publish.
+5. Sync bar repo (RemoveMissingFalseTestCase, RemoveMissingFalseTestCase).
+6. Assert that:
+    * when remove_missing=True, content of foo and bar is same
+    * when remove_missing=False, content of foo and bar differ
+
+"""
+
+from __future__ import unicode_literals
+
+try:  # try Python 3 import first
+    from urllib.parse import urljoin, urlparse
+except ImportError:
+    from urlparse import urljoin, urlparse  # pylint:disable=C0411,E0401
+
+import random
```

Comment:
Nitpick: This is a standard library import. Please place a few lines up, with the other standard lib imports.